hathor/cli/run_node.py

```diff
@@ -69,6 +69,7 @@ def create_parser(self) -> ArgumentParser:
                             'This is still a beta feature as it may cause issues when restarting the full node '
                             'after a crash.')
         parser.add_argument('--procname-prefix', help='Add a prefix to the process name', default='')
+        parser.add_argument('--sentry', help='Publish exception events to Sentry.')
```

Comment:
I prefer being more explicit in the help text saying that we 'Expect the Sentry DSN as parameter.'.

Reply:
I'm not too familiar with sentry, so maybe we should call it `--sentry-dsn` then.

Reply:
Done!